internal/validate/helpers.go

```diff
@@ -50,32 +50,33 @@ func GetPolicyConfig(ctx context.Context, policyConfiguration string) (string, e
 		if err != nil {
 			return "", err
 		}
-		return readPolicyConfigurationFile(ctx, configFile)
+		log.Debugf("Loading %s as policy configuration", policyConfiguration)
+		return ReadWorkspaceFile(ctx, configFile)
 	} else if source.SourceIsFile(policyConfiguration) && utils.HasJsonOrYamlExt(policyConfiguration) {
 		// If policyConfiguration is detected as a file and it has a json or yaml extension,
 		// we read its contents and return it.
-		return readPolicyConfigurationFile(ctx, policyConfiguration)
+		log.Debugf("Loading %s as policy configuration", policyConfiguration)
+		return ReadWorkspaceFile(ctx, policyConfiguration)
 	}
 
 	// If policyConfiguration is not a file path, git url, or https url,
 	// we assume it's a string and return it as is.
 	return policyConfiguration, nil
 }
 
-// Read policyConfiguration file and return its contents.
-func readPolicyConfigurationFile(ctx context.Context, policyConfiguration string) (string, error) {
-	// Check if policyConfig is a file path. If so, try to read it.
-	// If successful we write that into the data.policyConfiguration var.
+// Read file from the workspace and return its contents.
+func ReadWorkspaceFile(ctx context.Context, fileName string) (string, error) {
+	// Check if fileName is a file path. If so, try to read it.
```

Comment:
I know it's not exactly part of your patch here, but is this comment wrong now? Seems like we're just assuming it's a file path.

Reply:
makes sense. will get rid of it